Apps/Src/UpdateDisplay.c

```diff
@@ -306,6 +308,40 @@ UpdateDisplayError_t UpdateDisplay_SetAccel(uint8_t percent){
 	return ret;
 }
 
+UpdateDisplayError_t UpdateDisplay_SetForwardPower(uint8_t percent) {
+	static uint8_t lastPercentPower = 0;
+	if(percent == lastPercentPower){
+		return UPDATEDISPLAY_ERR_NO_CHANGE;
+	}
+
+	UpdateDisplayError_t ret = UpdateDisplay_SetComponent(MOTOR, percent);
+	assertUpdateDisplayError(ret);
+	
+	if(ret == UPDATEDISPLAY_ERR_NONE) lastPercentPower = percent;
+	return ret;
+}
+
+UpdateDisplayError_t UpdateDisplay_SetRegenPower(int32_t percent){
+	static uint8_t lastPercentRegen = 0;
+	if(percent == lastPercentRegen){
+		return UPDATEDISPLAY_ERR_NO_CHANGE;
+	}
+
+	UpdateDisplayError_t ret = UpdateDisplay_SetComponent(REGEN_METER, percent);
+	assertUpdateDisplayError(ret);
+	
+	if(ret == UPDATEDISPLAY_ERR_NONE) lastPercentRegen = percent;
+
+	// In Nextion, we are displaying regeneration slider by
+	// modifying the background percentage.
+	return (100 - ret);
+}
+
+void UpdateDisplay_SetBackEMF(uint8_t forwardPercent, int32_t regenPercent) {
+	UpdateDisplay_SetForwardPower(forwardPercent);
+	UpdateDisplay_SetRegenPower(regenPercent);
+}
```

Comment:
Maybe this is nitpicky, but if setting backemf is just a wrapper function for simultaneously setting forward and regen power, I'm not sure that's the best idea. I can definitely see someone trying to manually set forward power / regen power and it getting overwritten by a setbackemf call somewhere. 

@IshDeshpa any clarity on why we need a dedicated setter for this?  

If we do need it, can it maybe me named more specifically, or given a header comment noting that it shouldn't be used in conjunction with the other two calls. Could also deprecate the other two functions if we want everyone to use this moving forward.